### PR TITLE
Feature/621 User Profile

### PR DESCRIPTION
**Pull request by john-labbate**
User Profile > Save Changes Without Returning the User to the Search Results Screen
[#621](https://github.com/GSA/smartpay-training/issues/621)

### CODE REVIEW

File: training-front-end/src/components/AdminRepository.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<script>
+  import { profile} from '../stores/user'
```
Comment (john-labbate) [inline training-front-end/src/components/AdminRepository.vue:2]: Going forward, all server calls from the admin pages will be added to this file. Eventually the goal is to reduce duplicated code, enforce standardization on how we interact with the server, simplify front-end testing. This pattern is known as the front-end repository/service/api pattern.

File: training-front-end/src/components/AdminEditUserDetails.vue [hunk 2]
```diff
@@ -72,60 +71,29 @@ async function update_user_info() {
   is_saving.value = true
   show_spinner.value = true
 
-  // When user has choosen a bureau use that id instead of the agency
+  // When user has chosen a bureau use that id instead of the agency
```
Comment (CodyHinze) [inline training-front-end/src/components/AdminEditUserDetails.vue:74]: Thanks for cleaning up the repo.

File: training-front-end/src/components/AdminSearchUser.vue [hunk 3]
```diff
@@ -182,7 +152,7 @@
         <AdminUserSearchTable 
           :number-of-results="numberOfResults"
           :search-results="searchResults" 
-          @select-item="setCurrentUser"
+          @select-item="setSelectedUser"
```
Comment (CodyHinze) [inline training-front-end/src/components/AdminSearchUser.vue:155]: ❤️ much better name

File: training/api/api_v1/users.py [hunk 4]
```diff
@@ -80,13 +80,27 @@ def get_users(
     '''
     Get/users is used to search users for admin portal
     currently search only support search by user name and email address, searchText is required field.
-    It may have additional search criteira in future, which will require logic update.
+    It may have additional search criteria in future, which will require logic update.
```
Comment (CodyHinze) [inline training/api/api_v1/users.py:83]: Thanks for cleaning up.

**Review by CodyHinze — APPROVED** [review]
Looks great! Thanks for correcting typos in existing code comments.

**Pull request merged**